pkgs/top-level/all-packages.nix

```diff
@@ -16022,7 +16022,7 @@ with pkgs;
       ImageCaptureCore GSS ImageIO;
   };
 
-  emacsPackagesGen = emacs: self: let callPackage = newScope self; in rec {
+  emacsPackagesFor = emacs: self: let callPackage = newScope self; in rec {
```

Comment:
Does this need an alias.nix entry? Not sure whether this was intended to be user-exposed.